easybuild/easyconfigs/p/PASA/PASA-2.5.3-foss-2024a.eb

```
@@ -0,0 +1,57 @@
+easyblock = 'ConfigureMake'
+
+name = 'PASA'
+version = '2.5.3'
+
+homepage = 'https://github.com/PASApipeline/PASApipeline'
+description = """PASA, acronym for Program to Assemble Spliced Alignments (and pronounced 'pass-uh'),
+ is a eukaryotic genome annotation tool that exploits spliced alignments of expressed transcript
+ sequences to automatically model gene structures, and to maintain gene structure annotation consistent
+ with the most recently available experimental sequence data. PASA also identifies and classifies all
+ splicing variations supported by the transcript alignments."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+source_urls = ['https://github.com/PASApipeline/PASApipeline/archive/']
+sources = ['%(namelower)s-v%(version)s.tar.gz']
+checksums = ['9b2f6a301b73fd8c713af0977cf6e97f3d928988d6b79715ebe81e19b51152eb']
+
+dependencies = [
+    ('SQLite', '3.45.3'),
+    ('GMAP-GSNAP', '2024-09-18'),
+    ('pblat', '2.5.1'),
+    ('minimap2', '2.28'),
```

@laraPPr This should be `2.29` (cfr. easyconfig that was added in #22791)

```suggestion
    ('minimap2', '2.29'),